bot/src/main/java/com/intern/bot/client/TelegramClient.java

```diff
@@ -0,0 +1,29 @@
+package com.intern.bot.client;
+
+import com.intern.bot.service.TelegramBotService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.telegram.telegrambots.meta.api.objects.Message;
+import reactor.core.publisher.Mono;
+
+@RequiredArgsConstructor
+@Service
+public class TelegramClient {
+
+  final TelegramBotService telegramBotService;
+
+  public Mono<String> messageHandler(Message message) {
+    Mono<String> queryResult = null;
```

Comment:
It's not a good idea to return `null` when you return `Mono`. This is like assigning `Optional<String> query = null`. Poopoo coding. Assign it to `Mono.empty()` instead.